android/app/src/test/java/com/getcapacitor/myapp/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.getcapacitor.myapp;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
```

Comment:
remove?

Reply:
Sure ? We may use that in the future, all files under `android` are generated, but a few modifications I made to the `build.gradle`

Reply:
yes, I don't think we'll use any native test, probably more white box test like Webdriver.io